crates/runtime/src/http/v1.rs

```diff
@@ -166,12 +166,12 @@ pub(crate) mod query {
 
         match is_data_from_cache {
             Some(true) => {
-                if let Ok(value) = "HIT".parse() {
+                if let Ok(value) = "Hit from spiceai".parse() {
```

Comment:
This adds more clarity and matches format of industry leading systems, for example  AWS CloudFront

```
< HTTP/1.1 200 OK< Content-Type: text/html
< Content-Length: 437
...
< Server: AmazonS3
< Age: 14
**< X-Cache: Hit from cloudfront**
```